components/search_results_item.jsx

```diff
@@ -255,15 +253,13 @@ export default class SearchResultsItem extends React.Component {
             );
 
             message = (
-                <PostBodyAdditionalContent post={post}>
```

Comment:
Removed `PostBodyAdditionalContent ` here and it might remove quite a few things but the note(https://github.com/mattermost/mattermost-server/issues/7542) here says it should not have any preview in search results as well. Let me know if this works